packages/odoo/test/MockOdooWithSearch.test.js

```diff
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import { MockOdooWithSearch } from './MockOdooWithSearch';
+
+const mockData = () => [
+  {
+    id: 1,
+    name: 'A Ltd',
+    is_company: true,
+    im_status: 'im_partner',
+    email: 'a@example.com',
+  },
+  {
+    id: 2,
+    name: 'B Ltd',
+    is_company: true,
+    im_status: 'im_partner',
+    phone: '123',
+  },
+  { id: 3, name: 'C Sole', is_company: false, im_status: 'im_partner' },
+  { id: 4, name: 'D NGO', is_company: true, im_status: 'im_vendor' },
+  {
+    id: 5,
+    name: 'E Corp',
+    is_company: true,
+    im_status: 'im_partner',
+    city: 'Nairobi',
+  },
+];
+
+describe('MockOdooWithSearch.searchRead', () => {
+  let data, mock;
+
+  beforeEach(() => {
```

Comment:
Because the data doesn't change between tests, I think this is a `before`, not a `beforeEach()`